tagged-localbuild.xml

```diff
@@ -0,0 +1,825 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="NXP" fetch="https://github.com/NXP/"/>
+  <remote name="aosp" review="https://android-review.googlesource.com/" fetch="https://android.googlesource.com/"/>
+  <remote name="exfatprogs" fetch="https://github.com/exfatprogs/"/>
+  <remote name="hybris" fetch="https://github.com/mer-hybris"/>
+  <remote name="sony" fetch="https://github.com/sonyxperiadev/"/>
+
+  <default remote="aosp" revision="refs/tags/android-11.0.0_r43" sync-j="4"/>
+
+  <project name="QcRilAm" path="vendor/oss/qcrilam" remote="sony" revision="ef51ec65f5609e0fe9180ec3d2d054b72c0b8bee" upstream="master" dest-branch="master" groups="device"/>
+  <project name="SonyOpenTelephony" path="vendor/oss/opentelephony" remote="sony" revision="c0b6d4306cd69764de63d2064c99c1224b840430" upstream="master" dest-branch="master" groups="device"/>
+  <project name="camera" path="vendor/qcom/opensource/camera" remote="sony" revision="93520c5e63d30b4c35f1665aa9d644d37cb5491b" upstream="aosp/LA.UM.8.11.r1" dest-branch="aosp/LA.UM.8.11.r1" groups="device"/>
+  <project name="device-sony-akari" path="device/sony/akari" remote="sony" revision="ddbd49032d52184a7942eced2a5dc5f1b456058f" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-akatsuki" path="device/sony/akatsuki" remote="sony" revision="3fcb246ea543ce8d2e549b707a61b6c001594163" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-apollo" path="device/sony/apollo" remote="sony" revision="e702eb3698c31c2f71e90bfbb4558177b8bcfcb6" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-bahamut" path="device/sony/bahamut" remote="sony" revision="04bda77637e939bfbe0842ca9e2790395372df13" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-common" path="device/sony/common" remote="sony" revision="0d9406f37d886c7fcbc60e38823c5b20d975558d" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
```

Comment:
This repo has been force-pushed to (it happens often upstream during active dev).
I'm updating the adaptation locally, will comment here about results.

Reply:
I know, I just used non tagged manifest. Before everything is stable non tagged should be better anyway.

Reply:
Have you updated to android-11.0.0_r46 as well?:)

Reply:
I did a diff there are not really many changes in later releases for the hal side of things.

Reply:
Now you pushed an update to r48 instead, which Sony hasn't tested with yet, nor pushed out compat repos.